docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM phpdockerio/php72-fpm:latest
+FROM phpdockerio/php72-cli:latest
```

Comment:
Is there a reason to not use the oficial php image?

`FROM php:7.2-cli`

Reply:
I'll test here and see how it goes. I think that there's no reason not to.

Reply:
didn't work out of the box (just change it). it would be a little more complicated to debug this, and because it's only for development  work, I think it's not really a big deal this way for now